RepoLayout.props

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project>
```

Comment:
❔ Can we move this into an **eng** folder?

Reply:
https://github.com/gitextensions/gitextensions/pull/8522#discussion_r622826901